packages/website/src/pages/terms.js

```diff
@@ -0,0 +1,234 @@
+        under any circumstances other than to respond to subpoenas, court orders, or other legal process, as required by
+        law, or to establish or exercise our legal rights or defend against legal claims. We will never sell or share
+        any of this data.
+      </P>
+
+      <h2>2. License, Restrictions and Prohibited Acts</h2>
+      <P>
+        Grant of License. Skynet Labs grants you a limited, nonexclusive, nontransferable, revocable license to access
+        and make personal use of the Website and the Services in accordance with the terms set forth in the Terms of
+        Use. Skynet Labs reserves all right, title and interest not expressly granted under this license to the fullest
+        extent possible under applicable laws. You may not sublicense, assign, or transfer the license granted to you
+        under the Terms of Use, and any attempt to sublicense, assign, or transfer any part of your rights under the
+        Terms of Use is void.
+      </P>
+
+      <P>
+        <strong>Restrictions and Prohibited Acts.</strong> So that the Website is available for all users to enjoy, you
+        may not, and hereby represent, warrant and covenant that you will not or permit or enable third party to:
+      </P>
+      <P>
+        use the Website in any way that (i) violates the terms of the Terms of Use; (ii) is unlawful, harmful,
+        threatening, tortious, defamatory, libellous, abusive, obscene, invasive of another's privacy, hateful,
+        fraudulent or malicious; (iii) involves the transmission of "junk mail," "chain letters," or unsolicited mass
+        mailing or "spamming"; (iv) involves the sending of any virus, Trojan horse, worm, harmful code, shutdown
+        mechanism or similar mechanism; (v) interferes with or disrupts the Website or any server or network involved
+        with the operation of the Website; or (vi) otherwise violates any local, national or other applicable law or
+        regulation; use our Services to harm, in any way, the operation of any website that you do not own or operate,
+        including, without limitation, by generating recurring traffic to such website through the conduct of tests
+        using our Services; modify, publish, transmit, transfer or sell, reproduce, create derivative works from,
+        distribute, perform, display or in any way exploit any of the content of the Website in whole or in part, except
+        as expressly permitted; or use automated tools to operate the Services on the Website (scripts, etc.).
+        Sanctions. ANY USE OF THE WEBSITE NOT SPECIFICALLY PERMITTED UNDER THE TERMS OF USE IS STRICTLY PROHIBITED AND
+        MAY RESULT, AT SKYNET LABS’ DISCRETION, IN THE DELETION OF YOUR DATA AND/OR THE BLACKLISTING OF YOUR IP ADDRESS.
+      </P>
+      <P>
+        <strong>Contact.</strong> Any misuse of the Services may be reported to{" "}
+        <a className="text-primary" href="mailto:report@siasky.net">
```

Comment:
this should be coming from the environment I think

Reply:
@peterjan Yeah, these were hardcoded on the website recently (#1843), so I left them hardcoded here as well.

Are there env variables I can rely on for this?